.gitignore

```diff
@@ -1,21 +1,108 @@
-*.egg-info
-*.pyc
-*.pyo
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
 *$py.class
+
+# C extensions
 *.so
-__pycache__
-.coverage
-.installed.cfg
-.mr.developer.cfg
-nosetests.xml
-coverage.xml
-.eggs/
-.tox/
-bin/
+
+# Distribution / packaging
+.Python
 build/
-cover/
-eggs/
 develop-eggs/
 dist/
-docs/_build/
+downloads/
+eggs/
+.eggs/
+lib/
+lib64/
 parts/
+sdist/
+var/
+wheels/
+*.egg-info/
+.installed.cfg
+*.egg
+MANIFEST
+
+# PyInstaller
+#  Usually these files are written by a python script from a template
+#  before PyInstaller builds the exe, so as to inject date/other infos into it.
+*.manifest
+*.spec
+
+# Installer logs
+pip-log.txt
+pip-delete-this-directory.txt
+
+# Unit test / coverage reports
+htmlcov/
+.tox/
+.coverage
+.coverage.*
+.cache
+nosetests.xml
+coverage.xml
+*.cover
+.hypothesis/
+
+# Translations
+*.mo
+*.pot
+
+# Django stuff:
+*.log
+.static_storage/
+.media/
+local_settings.py
+
+# Flask stuff:
+instance/
+.webassets-cache
+
+# Scrapy stuff:
+.scrapy
+
+# Sphinx documentation
+docs/_build/
+
+# PyBuilder
+target/
+
+# Jupyter Notebook
+.ipynb_checkpoints
+
+# pyenv
+.python-version
+
+# celery beat schedule file
+celerybeat-schedule
+
+# SageMath parsed files
+*.sage.py
+
+# Environments
+.env
+.venv
+env/
+venv/
+ENV/
+env.bak/
+venv.bak/
+
+# Spyder project settings
+.spyderproject
+.spyproject
+
+# Rope project settings
+.ropeproject
+
+# mkdocs documentation
+/site
+
+# mypy
+.mypy_cache/
+
+# dataflake.fakeldap
```

Comment:
these comments are referring to what generates the files. *everything* here is for `dataflake.fakeldap`. [mr.developer](https://pypi.python.org/pypi/mr.developer) is what generates `.mr.developer.cfg`, and that is what the section header should be.